src/main/scala/nl.knaw.dans.easy.sword2/SwordConfig.scala

```diff
@@ -19,7 +19,7 @@ import org.swordapp.server.SwordConfigurationDefault
 
 class SwordConfig extends SwordConfigurationDefault {
   override def getAuthType: String = "Basic"
-
+  override def returnStackTraceInError(): Boolean = false //TODO would it be desirable to do this with the application.properties file?
```

Comment:
@janvanmansum @lindareijnhoudt what do you think?

On the one hand I don't think it's a bad idea to put it in `application.properties`: we may want to have these stacktraces on our development machines and not on the ACT/production machines.

On the other hand, I do value the consistency of output between the different servers, which is broken when we configure them differently.